Readme merge from discord text file

cychow opened the pull request:
Adding README.md based off of Jack's ideas text file

fenixfurion committed:
add README

cychow committed:
fix spacing for usage

jackhorton commented on README.md:
```diff
@@ -0,0 +1,39 @@
+
+
+
+
```
```suggestion
## Proposed SQL Schema
```

jackhorton commented on README.md:
```diff
@@ -0,0 +1,39 @@
+
+
+
+### Custom images
```
```suggestion
## Custom images
```

jackhorton commented on README.md:
```diff
@@ -0,0 +1,39 @@
+## v1 - basic functionality
```
```suggestion
# Roadmap

## v1 - basic functionality
```

jackhorton reviewed (approved):
Looks good overall, just some minor suggestions

cychow committed:
Apply suggestions from code review

Make v1 header 1 and add roadmap to indicate where the project is. Remove excessive whitespacing

Co-authored-by: Jack Horton <jackpeterhorton@gmail.com>

cychow reviewed (commented):
Suggestions seem fine; it was your ideas file in the first place
